### DISCUSSION TITLE
CompreFace Docker image in Mac M1 Chips

### DISCUSSION THREAD

**Discussion by naoa18-3** [exadel-inc/CompreFace#1042] [General]
Hi,
Do you guys know how to execute container in M1 Chip, versión 1.0.1. 
Is this possible? 

Thank's in advance!

**Comment by pospielov** [comment]
M1 chip is based on ARM. 
As for now, CompreFace doesn't support ARM CPUs.
We have some progress in this. Our master branch can already be built for ARM devices. 
But this version has memory leaks, so we can't release it as is. We will continue to work on it.

**Reply by NMGAndy** [reply]
Hi! Any News?

**Comment by rajdey1** [comment]
I’m currently running Compreface on a system with an AArch64 (ARM 64-bit) architecture and encountered compatibility issues with some services. Specifically, I am having trouble running the compreface-core service on this architecture.

I’ve successfully managed to run other containers on my AArch64 setup by using multi-platform support via docker buildx, but I am unsure if compreface-core is compatible or if there are any known issues or requirements for running it on ARM-based systems.

Could you please provide information on the following:

AArch64 Support: Does Compreface officially support ARM 64-bit architecture? Are there specific steps or configurations required to run compreface-core on an AArch64 system?

Docker Images: Are there pre-built Docker images for ARM 64-bit architecture for Compreface services? If not, are there recommended practices or solutions for building these images from source?

Known Issues: Are there any known issues or limitations when running Compreface services on ARM-based systems?

Documentation: Is there any additional documentation or community resources for running Compreface on non-x86 architectures?

Any guidance or assistance would be greatly appreciated as it will help in ensuring compatibility and smooth operation of Compreface on ARM-based systems.

Thank you for your time and support!
